src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -121,129 +154,26 @@ const ApplicationModal = ({
+            <Button
+              key="submitR"
+              onClick={() => rejectApplication('reject')}
+              danger
+            >
               Reject
             </Button>,
           ]}
         >
           {currentApplication.role_name === 'mentee' ? (
-            <List size="small" bordered>
-              <List.Item>
-                <b>Role:</b>
-                {currentApplication.accepting_new_mentees === undefined
-                  ? 'Mentee'
-                  : 'Mentor'}
-              </List.Item>
-              <List.Item>
-                <b>Email:</b> {currentApplication.email}
-              </List.Item>
-              <List.Item>
-                <b>Location:</b> {currentApplication.city},{' '}
-                {currentApplication.state} {currentApplication.country}
-              </List.Item>
-              <List.Item>
-                <b>Membership Criteria:</b>
-                <ul>
-                  {currentApplication.formerly_incarcerated === true ? (
-                    <li>Formerly Incarcerated</li>
-                  ) : null}
-                  {currentApplication.low_income === true ? (
-                    <li>Low Income</li>
-                  ) : null}
-                  {currentApplication.underrepresented_group === true ? (
-                    <li>Belongs to underrepresented group</li>
-                  ) : null}
-                </ul>
-              </List.Item>
-              <List.Item>
-                {' '}
-                <b>Convictions:</b>{' '}
-                {`${
-                  currentApplication.formerly_incarcerated === true
-                    ? currentApplication.convictions
-                    : 'none'
-                }`}
-              </List.Item>
-              <List.Item>
-                <b>Applicant needs help with:</b>{' '}
-                <ul>
-                  {currentApplication.industry_knowledge === true ? (
-                    <li>Industry Knowledge</li>
-                  ) : null}
-                  {currentApplication.pair_programming === true ? (
-                    <li>Pair Programming</li>
-                  ) : null}
-                  {currentApplication.job_help === true ? (
-                    <li>Job Help</li>
-                  ) : null}
-                </ul>
-              </List.Item>
-              <List.Item>
-                <b>Subject most interested in:</b> {currentApplication.subject}
-              </List.Item>
-              <List.Item>
-                <b>Other information:</b> {currentApplication.other_info}
-              </List.Item>
-              <List.Item>
-                <b>Submission Date:</b>{' '}
-                {currentApplication.created_at.slice(0, 10)}
-              </List.Item>
-              <List.Item>
-                <b>Application Status:</b> {currentApplication.other_info}
-              </List.Item>
-            </List>
+            <MenteeModal applicant={currentApplication} />
```

Comment:
Great work cutting down on lines in this file by moving this application to its own component!